### PR TITLE
[TACHYON-1428] Documented parts of the tachyon.client package in preparation for the checkstyle rule

### PR DESCRIPTION
**Pull request by HorizonNet**
https://tachyon.atlassian.net/browse/TACHYON-1428

### CODE REVIEW

File: clients/unshaded/src/main/java/tachyon/client/file/TachyonFileSystem.java [hunk 1]
```diff
@@ -210,6 +263,12 @@ public FileOutStream getOutStream(long fileId, OutStreamOptions options) throws
   /**
    * Convenience method for {@link #loadMetadata(TachyonURI, LoadMetadataOptions)} with default
    * options.
+   *
+   * @param path the path for which to load metadat from UFS
```
Comment (gpang) [inline clients/unshaded/src/main/java/tachyon/client/file/TachyonFileSystem.java]: metadat -> metadata

File: clients/unshaded/src/main/java/tachyon/client/lineage/LineageMasterClient.java [hunk 2]
```diff
@@ -71,6 +71,16 @@ protected void afterConnect() throws IOException {
     mClient = new LineageMasterClientService.Client(mProtocol);
   }
 
+  /**
+   * Creates a lineage.
+   *
+   * @param inputFiles the list of input files
```
Comment (gpang) [inline clients/unshaded/src/main/java/tachyon/client/lineage/LineageMasterClient.java]: "file names"

File: clients/unshaded/src/main/java/tachyon/client/lineage/LineageMasterClient.java [hunk 3]
```diff
@@ -71,6 +71,16 @@ protected void afterConnect() throws IOException {
...
+   * @param outputFiles the list of output files
```
Comment (gpang) [inline clients/unshaded/src/main/java/tachyon/client/lineage/LineageMasterClient.java]: "file names"

File: clients/unshaded/src/main/java/tachyon/client/lineage/LineageMasterClient.java [hunk 4]
```diff
@@ -93,6 +112,16 @@ public Boolean call() throws TachyonTException, TException {
     });
   }
 
+  /**
+   * Reinilizates a given file.
```
Comment (gpang) [inline clients/unshaded/src/main/java/tachyon/client/lineage/LineageMasterClient.java]: Reinilizates -> Reinitializates

**Comment by gpang** [comment]
Thanks for the PR! I left a few comments.

**Comment by HorizonNet** [comment]
@gpang Thanks for the comments. I updated the PR accordingly.

**Comment by apc999** [comment]
+1 LGTM. Thanks for the improvement!

File: clients/unshaded/src/main/java/tachyon/client/lineage/LineageMasterClient.java [hunk 5]
```diff
@@ -103,6 +132,13 @@ public Long call() throws TachyonTException, TException {
     });
   }
 
+  /**
+   * Retrieve the list of lineage information.
```
Comment (yupeng9) [inline clients/unshaded/src/main/java/tachyon/client/lineage/LineageMasterClient.java]: Retrieve -> Retrieves

**Comment by yupeng9** [comment]
Thanks for the doc improvement. I left one minor comment.

**Comment by HorizonNet** [comment]
@yupeng9 Thanks for the comment. I just updated the PR.

**Comment by yupeng9** [comment]
Thanks! LGTM

**Pull request merged**